Dockerfile/electrs/Dockerfile

```diff
@@ -62,4 +62,4 @@ EXPOSE 4222
 
 STOPSIGNAL SIGINT
 
-ENTRYPOINT ["/bin/bash", "-c", "/usr/local/bin/electrs -vvvv --daemon-dir $BITCOIN_DATA --db-dir $ELECTRS_DATA --conf $ELECTRS_CONF "]
+ENTRYPOINT ["/bin/bash", "-c", "/usr/local/bin/electrs -vvvv --daemon-dir $BITCOIN_DATA --db-dir $ELECTRS_DATA"]
```

Comment:
Why do we need "/bin/bash", while Bitcoin Core can be launched w/o it?

Reply:
when using this image we don't need to add extra flags via the command option, since a way to change most of the configuration options is already provided (via env variables in the form of `ELECTRS_`)